Source/VMware.vSphereDSC/Tests/Integration/VMHostVssSecurity.Integration.Tests.ps1

```diff
@@ -0,0 +1,170 @@
+<#
```

Comment:
The same comments as for the VMHostVss Integration Tests.

Reply:
Changed